.husky/hooks/pre-push

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "$(git status --short)" ]]; then
+    git stash
+    function unstash() {
+        git reset --hard
+        git stash pop
+    }
+    trap unstash EXIT
+fi
+
+make manifests generate test
+
+git diff --exit-code --quiet || (git status && exit 1)
```

Comment:
If the intent is to see what's changed you can use the first command with extra flags like:
- `--stat` (you can specify what you want to see)
- `--shortstat`
- `--name-status`

Reply:
Meaning you can drop `|| (git status && exit 1)`